Sources/PrimerSDK/Classes/Data Models/PrimerSettings.swift

```diff
@@ -1,6 +1,6 @@
 #if canImport(UIKit)
 
-public typealias ClientTokenCallBack = (_ completionHandler: @escaping (Result<CreateClientTokenResponse, Error>) -> Void) -> Void
+public typealias ClientTokenCallBack = (_ completionHandler: @escaping (Result<String, Error>) -> Void) -> Void
```

Comment:
down the line, maybe we wrap this in a struct something? (keeping in mind the feedback regarding decodable etc.). Seems like an easy way to introduce bugs passing an anonymous strings around.

Reply:
Well, it's a very usual practice to pass primitive vars if possible. This is what Mayur wanted, to not have to create a custom decodable object just as a string wrapper. Moreover, now we don't even have the expiration timestamp in this object, so it'd be a struct containing just a string.